src/test/java/org/kohsuke/github/AbstractGHAppInstallationTest.java

```diff
@@ -63,15 +83,15 @@ private GHAppInstallation getAppInstallationWithToken(String jwtToken) throws IO
     }
 
     protected GHAppInstallation getAppInstallationWithTokenApp1() throws IOException {
-        return getAppInstallationWithToken(createJwtToken(PRIVATE_KEY_FILE_APP_1, TEST_APP_ID_1));
+        return getAppInstallationWithToken(JWT_PROVIDER_1.getEncodedAuthorization());
     }
 
     protected GHAppInstallation getAppInstallationWithTokenApp2() throws IOException {
-        return getAppInstallationWithToken(createJwtToken(PRIVATE_KEY_FILE_APP_2, TEST_APP_ID_2));
+        return getAppInstallationWithToken(JWT_PROVIDER_2.getEncodedAuthorization());
     }
 
     protected GHAppInstallation getAppInstallationWithTokenApp3() throws IOException {
-        return getAppInstallationWithToken(createJwtToken(PRIVATE_KEY_FILE_APP_3, TEST_APP_ID_3));
+        return getAppInstallationWithToken(JWT_PROVIDER_3.getEncodedAuthorization());
```

Comment:
@MarcosCela 
To test JWT provider, maybe just add an assertion to the test that uses these.

Reply:
Tests added, what do you think?